rails-i18n.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = '>= 1.8.11'
 
   s.add_runtime_dependency('i18n', '>= 0.7', '< 2')
-  s.add_runtime_dependency('railties', '>= 6.0.0', '< 7')
+  s.add_runtime_dependency('railties', '>= 7.0.0', '< 8')
```

Comment:
Since everything still works with rails 6, I think we should only relax the upper and not restrict the lower one i.e.

```ruby
s.add_runtime_dependency('railties', '>= 6.0.0', '< 8')
```

Reply:
Agreed, supporting both Rails 6 and 7 will help with transitioning.

Reply:
@f6p can you please do this small change?

Reply:
I think it should just be `>= 6` so that it's compatible with future versions of rails in an "evergreen" setup